docs/DeveloperGuide.md

```diff
@@ -250,44 +263,66 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* * *`  | student                                       | add an tuition appointment                                                                    |                                                                                           |
+| `* * *`  | student                                       | view my tuition appointment                                                                   | view the details of the tuition appointment (i.e., tutor's name, location, date and time) |
+| `* * *`  | student                                       | delete a tuition appointment                                                                  | know that the tuition appointment is cancelled                                            |
+| `* * *`  | student                                       | check my personal tuition appointment schedule                                                | know the timing of all my tuition appointments                                            |
+| `* * *`  | student                                       | filter tutors by their subject discipline                                                     | find a tutor that caters to my needs (academic)                                           |
+| `* * *`  | student                                       | filter tutor by cost                                                                          | find a tutor that fits into budget as well                                                |
+| `* * *`  | student                                       | filter a tutor by his/her name                                                                | view his/her profile                                                                      |
+| `* * *`  | student                                       | filter tutors by their years of experience                                                    | find a tutor with experience within the range of my expectation                           |
+| `* * *`  | student                                       | filter tutors by their available timeslots                                                    | find a tutor with matched tuition time                                                    |
+| `* * *`  | student                                       | filter tutors by their available location                                                     | find tutors in a specific area                                                            |
 *{More to be added}*
 
 ### Use cases
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
-
-**MSS**
-
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
-
+### Appointment
+<pre>
+Use Case UC001: Add an appointment
```

Comment:
Is this use case not completed yet?

Reply:
This meant to serve as a base template as I modified the styling slightly. Can choose to adopt this styling or go with the default style from AB3.